cmd/limactl/template.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/lima-vm/lima/pkg/limatmpl"
+	"github.com/spf13/cobra"
+)
+
+func newTemplateCommand() *cobra.Command {
+	templateCommand := &cobra.Command{
+		Use:           "template",
+		Aliases:       []string{"tmpl"},
+		Short:         "Lima template management",
+		SilenceUsage:  true,
+		SilenceErrors: true,
+		GroupID:       advancedCommand,
+		Hidden:        true,
+	}
+	templateCommand.AddCommand(
+		newTemplateCopyCommand(),
+		newTemplateValidateCommand(),
+	)
+	return templateCommand
+}
+
+func newTemplateCopyCommand() *cobra.Command {
+	templateCopyCommand := &cobra.Command{
+		Use:   "copy TEMPLATE DEST",
+		Short: "Copy template",
```

Comment:
Maybe add `Long:` to explain the detailed usage and the motivation

Reply:
ping @jandubois

Reply:
Sorry, I was busy this week with other things; PTAL